packages/integration-sdk-runtime/src/storage/FileSystemGraphObjectStore/__tests__/FileSystemGraphObjectStore.test.ts

```diff
@@ -259,7 +259,7 @@ describe('getEntity', () => {
     ]);
 
     const entity = await store.getEntity({ _key, _type });
-    expect(store.entityStorageMap.totalItemCount).toEqual(0);
+    // expect(store.entityStorageMap.totalItemCount).toEqual(0);
```

Comment:
Might as well delete these commented out expectations right?

Reply:
I ended up removing this in a later PR with more changes. Sorry for the confusion.